axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/AxonServerEventStore.java

```diff
@@ -475,7 +475,7 @@ protected void storeSnapshot(DomainEventMessage<?> snapshot, Serializer serializ
             EventChannel eventChannel = connectionManager.getConnection(context).eventChannel();
 
             AggregateEventStream aggregateStream;
-            if (firstSequenceNumber > 0) {
+            if (firstSequenceNumber >= 0) {
                 aggregateStream = eventChannel.openAggregateStream(aggregateIdentifier, firstSequenceNumber);
             } else if (firstSequenceNumber == ALLOW_SNAPSHOTS_MAGIC_VALUE && !snapshotFilterSet) {
                 aggregateStream = eventChannel.openAggregateStream(aggregateIdentifier, true);
```

Comment:
The third else calls   eventChannel.openAggregateStream(aggregateIdentifier)  wich also calls 
.openAggregateStream(aggregateIdentifier,true)  , so what's the use of this branche ?

Reply:
just for overriding ?

Reply:
I was actually going to explain every branch, and then I indeed followed up your point, noting the javadoc of the `EventChannel` as well that basically line 481 and 483 do exactly the same. Thanks for noting, I am pretty confident the `else` should invoke `openAggregateStream(aggregateIdentifier, false)` instead of the default which uses true. Thanks for signaling this!

Reply:
ah, ok.